OSDOCS-15948# RN for for enabling token-based authentication on an existing AWS cluster (#100303)
openshift/openshift-docs: release_notes/ocp-4-20-release-notes.adoc

[id="ocp-release-notes-postinstallation-configuration_{context}"]
=== Postinstallation configuration

[id="ocp-release-notes-enabling-sts-existing-cluster_{context}"]
==== Enabling {aws-full} {sts-first} on an existing cluster

> With this release, you can configure your {aws-short} {product-title} cluster to use {sts-short} if you did not do so at installation.

Reviewer: ```suggestion
With this release, you can configure your {aws-short} {product-title} cluster to use {sts-short} even if you did not configure during installation.
```

Alternatively - 

`With this release, you can configure your {aws-short} {product-title} cluster to use {sts-short}  post-installation if you did not configure during installation.`

Small nit to consider.

Author: @xenolinux could you explain your reasoning?

Author: @xenolinux I updated as per your suggestions. This is back to you to complete the merge (I don't have merge privileges).
